WcaOnRails/app/views/competitions/index.html.erb

```diff
@@ -1,17 +1,139 @@
 <% provide(:title, 'Competitions') %>
 
 <div class="container">
-  <h1><%= yield(:title) %></h1>
-
-  <%= link_to "New competition", new_competition_path %>
-  <table class="table">
-    <% @competitions.each do |competition| %>
-      <tr>
-        <td><%= competition.id %></td>
-        <td><%= competition.name %></td>
-        <td><%= link_to "Edit", admin_edit_competition_path(competition) %></td>
-        <td><%= link_to "Results", "/results/competition.php?competitionId=#{competition.id}" %></td>
-      </tr>
+  <h2><%= yield(:title) %></h2>
+
+  <div id="scroller-anchor"></div>
+  <div id="comp-query" class="col-md-12">
+    <% if params[:display] == "List" %>
+      <ul>
```

Comment:
Ah - those seem like good ideas. Since the behavior might not be clear from the outset without text, how about tooltips as well? Maybe greyed out icons (with appropriate tooltips) for when there are no competitions.  Also how about having them in chronological order (past, current, future)?